### PR TITLE
Splitting common code from netmesh proto

### PR DESCRIPTION
**Pull request by sbezverk**
This PR separates common proto definition into a separate common.proto file for convenience of using these definitions in other proto files.

Signed-off-by: Serguei Bezverkhi <sbezverk@cisco.com>

### CODE REVIEW

File: scripts/update-codegen.sh [hunk 1]
```diff
@@ -18,9 +18,9 @@ echo "Calling ${CODEGEN_PKG}/generate-groups.sh"
 
 echo "Generating other deepcopy funcs"
 "${GOPATH}"/bin/deepcopy-gen \
-  --input-dirs ./netmesh/model/netmesh \
+  --input-dirs ./netmesh/model/netmesh ./pkg/nsm/apis/common \
```
Comment (sttts) [inline scripts/update-codegen.sh]: the list syntax is not correct. It takes the second value as a normal command argument. You need another `--input-dirs` if I remember right. Same below.

File: netmesh/model/netmesh/netmesh.proto [hunk 2]
```diff
@@ -21,26 +21,21 @@ syntax = "proto3";
 
 package netmesh;
 
-message Label {
-    map<string ,string> selector = 1;
-}
+import "github.com/ligato/networkservicemesh/pkg/nsm/apis/common/common.proto";
```
Comment (mestery) [inline netmesh/model/netmesh/netmesh.proto]: I see you're doing the client API in pkg/nsm, should everything under netmesh/model/netmesh be moved into the pkg directory as well? If so, I'd do a separate commit with that move rather than combine it here. We can merge this one first.
